components/org.wso2.carbon.identity.context.rewrite.valve/src/main/java/org/wso2/carbon/identity/context/rewrite/bean/RewriteContext.java

```diff
@@ -54,13 +57,18 @@ public void setContext(String context) {
         this.context = context;
     }
 
-    public Pattern getPattern() {
```

Comment:
Let's not remove the API. Rather deprecate it. 
It's a fact that this API would not have used by any party, still let's not break it.
We can release a minor version later and remove the API

Reply:
Having the deprecated annotation should specify,
- from when it's deprecated
- what to use